app/collections/CourseInstances.coffee

```diff
@@ -10,3 +10,9 @@ module.exports = class CourseInstances extends CocoCollection
     options.data ?= {}
     options.data.ownerID = ownerID
     @fetch(options)
+
+  fetchForClassroom: (classroomID, options={}) ->
+    classroomID = classroomID.id or classroomID # handle if they pass in a user
```

Comment:
Generally these functions should be able to take models or strings and handle either, so when you're writing a view you don't have to worry about it. Probably should make it into a well-named function rather than relying on comments to explain what's going on.